src/app/shared/image/image-upload.component.ts

```diff
@@ -98,6 +99,9 @@ export class ImageUploadComponent implements DoCheck {
         if (this.browserImage.width < this.minWidth || this.browserImage.height < this.minHeight) {
           this.imgError = `The image provided is only ${this.browserImage.width} x ${this.browserImage.height} px
                            but should be at least ${this.minWidth} x ${this.minHeight} px.`;
+        } else if (this.square && this.browserImage.width !== this.browserImage.height) {
+          this.imgError = `Image width and height must be the same, but the image provided is 
+                           ${this.browserImage.width} x ${this.browserImage.height} px.`;
         } else {
```

Comment:
I dropped the "you" from "image you provided". (Not sure it's still in there, but Apple's human interface styleguide on error messages used to say "Avoid using “you,” “your,” “me,” and “my” as much as possible. Sometimes, text that identifies people directly can be ambiguous and can even be interpreted as an insult.") Changing up the wording seems to help with give a better indicator that this is a different error than the minimum size requirements. Back to you @gcampo88